xmlChart.go

```diff
@@ -380,6 +380,7 @@ type cChartLines struct {
 // cScaling directly maps the scaling element. This element contains
 // additional axis settings.
 type cScaling struct {
+	LogBase     *attrValFloat  `xml:"logBase"`
```

Comment:
I think the data type of `logBase` in the Go language should be int at here.

Reply:
But the according to [OOXMLSchemas](https://github.com/sc34wg4/OOXMLSchemas/blob/2b074ca2c5df38b18ac118646b329b508b5bdecc/Part1/OfficeOpenXML-XMLSchema-Strict/dml-chart.xsd#L1142-L1147), it's `xsd:double` and Excel does support the log base with a float number (you can see one of my test case with logbase 10.5 and the output XLSX file), so I think attrVarFloat should be used here.